Sources/Orbit/Components/Badge.swift

```diff
@@ -32,17 +32,17 @@ import SwiftUI
 /// When the provided content is empty, the component results in `EmptyView` so that it does not take up any space in the layout.
 ///
 /// - Note: [Orbit.kiwi documentation](https://orbit.kiwi/components/badge/)
-public struct Badge<LeadingIcon: View, TrailingIcon: View>: View, PotentiallyEmptyView {
+public struct Badge<LeadingIcon: View, Label: View, TrailingIcon: View>: View, PotentiallyEmptyView {
 
     @Environment(\.backgroundShape) private var backgroundShape
     @Environment(\.sizeCategory) private var sizeCategory
     @Environment(\.status) private var status
     @Environment(\.textColor) private var textColor
 
-    private let label: String
-    private let leadingIcon: LeadingIcon
-    private let trailingIcon: TrailingIcon
-    private let type: BadgeType
+    @ViewBuilder private let leadingIcon: LeadingIcon
+    @ViewBuilder private let label: Label
+    @ViewBuilder private let trailingIcon: TrailingIcon
+    @ViewBuilder private let type: BadgeType
```

Comment:
Is there a point to `@ViewBuilder` here if it's also used in the public init?

Reply:
It just makes it easier to distinguish "subcomponent views" from other non-view properties on a first glance. 

It would be technically required, but only in case some of our inits used viewbuilder syntax to initialize the content manually based on init parameters.